### PR TITLE
Edit ReadMe, Developer Guide and About Us

### PR DESCRIPTION
**Pull request by prit3010**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -3,15 +3,15 @@ layout: page
 title: AddressBook Level-3
```
Comment (kevinchangjk) [inline docs/index.md:3]: Mind changing this to "Coydir" too?

**Review by kevinchangjk — CHANGES_REQUESTED** [review]
One minor change required, otherwise LGTM.

File: docs/index.md (hunk 1 above)
Comment (prit3010) [reply docs/index.md]: Resolved

**Comment by ngshijun** [comment]
LGTM

**Pull request merged**